frontend/pom.xml

```diff
@@ -85,6 +85,9 @@
               <arguments>
                 <argument>${npm.cli.script}</argument>
                 <argument>install</argument>
+                <argument>--cache-min</argument>
+                <!-- in seconds -->
+                <argument>999999</argument>
```

Comment:
Are there any potential drawbacks with long cache timeouts?  In Maven, over-caching should only lead to using old SNAPSHOTs as far as I know, but I'm not sure if the JavaScript dependencies make such a distinction (release vs snapshot).